postgres/osfamilymap.yaml

```
@@ -62,6 +62,10 @@ RedHat:
     gpgkey: 'https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-{{ release }}'
     baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-$releasever-$basearch'
 
+{%- if 'osfinger' in grains and grains.osfinger in ('CentOS Linux-7', 'CentOS Linux-8') %}
```

I think the best way is to use `osmajorrelease` here to cover RHEL distros and other derivatives as well.
```suggestion
{%- if grains.get('osmaorrelease') == 7 %}
```

---

Good idea.